explorer/main.cpp

```diff
@@ -123,9 +128,8 @@ auto ExplorerMain(int argc, char** argv, void* static_for_main_addr,
     llvm::outs() << "result: " << *result << "\n";
     return EXIT_SUCCESS;
   } else {
-    llvm::errs() << result.error() << "\n";
+    llvm::errs() << "Execution error: " << result.error() << "\n";
```

Comment:
```suggestion
    llvm::errs() << result.error() << "\n";
```

The returned error will already be labeled, so this would be duplicative.